frontend/src/scenes/session-recordings/player/snapshot-processing/DecompressionWorkerManager.ts

```diff
@@ -273,7 +232,7 @@ export class DecompressionWorkerManager {
     private updateStats(
         duration: number,
         dataSize: number,
-        yieldDuration?: number,
+        _yieldDuration?: number,
```

Comment:
Why leave this parameter here? Seems we're passing undefined to it in both places where this method is called?